woocommerce/woocommerce-ios
# Update woocommerce.com to woo.com in URLs

Closes: #11181 


## Description


Since we've migrated from woocommerce.com to woo.com, we should update the URLs in the app to point to the new domain, rather than go through the redirect.

This PR updates all URLs in the app to the new domain.

## Testing instructions


They're everywhere, and it doesn't seem worthwhile to check every link through the app. As I've gone through, I've loaded them all in a web browser to make sure they all go to the right place. I'd recommend doing a spot check of some of the changes, and checking some in the app as well.

---
